Buildings/Resources/Scripts/BuildingsPy/conf.json

```diff
@@ -189,5 +189,32 @@
          "rtol": 1E-8
     },
     "model_name": "Buildings.Examples.DualFanDualDuct.ClosedLoop"
+  },
+  {
+    "jmodelica": {
+      "translate": false
+    },
+    "optimica": {
+      "translate": false
+    },
+    "model_name": "Buildings.Experimental.DHC.Examples.Combined.Generation5.Examples.ParallelConstantFlow"
+  },
+  {
+    "jmodelica": {
+      "translate": false
+    },
+    "optimica": {
+      "translate": false
+    },
+    "model_name": "Buildings.Experimental.DHC.Examples.Combined.Generation5.Examples.SeriesConstantFlow"
+  },
+  {
+    "jmodelica": {
+      "translate": false
+    },
+    "optimica": {
+      "translate": false
+    },
```

Comment:
Let's check why they don't translate on Optimica. I don't see why they should not work, other than maybe the use of the `when terminal() ...` construct.

Reply:
Indeed the `when terminal()` statement causes an error with OCT. Do you know a workaround?

Nevertheless, the two issues attached remain.
[Modelon#2021022539000651.pdf](https://github.com/lbl-srg/modelica-buildings/files/6260827/Modelon.2021022539000651.pdf)
[Modelon#2021022539000525.pdf](https://github.com/lbl-srg/modelica-buildings/files/6260828/Modelon.2021022539000525.pdf)

Reply:
As the body in the `when terminal()` construct only prints a diagnostics, we could either delete it, or replace `terminal()` with `initial()`. 

For the remaining issues in the pdf, can you please open an issue, and add in `conf.json` a comment at the respective models such as in
```
    "optimica": {
      "simulate": false,
      "comment": "See https://github.com/lbl-srg/modelica-buildings/issues/2231"
    },
```
This way we can track the exclusion back to the ticket and test the ticket with the next OCT release.

Reply:
Addressed.